sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwapExamples.cs

```diff
@@ -20,7 +20,7 @@ protected override void CreateAndUpsertMarketDataToLusid(string scope, ModelSele
             // For LUSID to pick up these quotes, we have added a RIC rule to the recipe (see BuildRecipeRequest in TestDataUtilities.cs) 
             // The RIC rule has a large quote interval, this means that we can use one reset quote for all the resets.
             // For accurate pricing, one would want to upsert a quote per reset. 
-            var quoteRequest = TestDataUtilities.BuildQuoteRequest("USD6M", QuoteSeriesId.InstrumentIdTypeEnum.ClientInternal, 0.05m, "USD", TestDataUtilities.EffectiveAt);
+            var quoteRequest = TestDataUtilities.BuildQuoteRequest("BP00", QuoteSeriesId.InstrumentIdTypeEnum.RIC, 0.05m, "USD", TestDataUtilities.EffectiveAt);
```

Comment:
This change is so the built quote has identifiers that match what is needed to be a reset for the standard example of a swap. It is likely the reason this worked before, and didn't break any other tests by changing, is because no valuation requests on swaps required a reset. i.e. all our valuations were purely forward looking and didn't require any past data.

Reply:
Am little surprised the existing test (before your changes) was purely forward looking and didn't require reset. I recall hacking at this... I wonder if it is because we've changed the underlying swap and/or effectiveAt since my merge.... I'm wondering if "Vanilla" swap refers to a different swap to the one I used....

Reply:
what is the corresponding change in the recipe for changing the quote provider type? where does a user see this?

Reply:
@jz-finbourne This no longer uses Nick's refactor, so there is no concept of "Vanilla" or otherwise yet. At the moment there is just the one standard swap.
It looks like valuation calls are generically being made effective 2020-01-02, but the instrument's start date is 2020-02-07, so no resets required.

Reply:
@rizwansaeed There is no change needed in the recipe. This quote was not actually being used in the tests. 
Currently, we set up an extremely generic recipe using a `CreateAndUpsertRecipe` method. This already contains a rule that says we can look back up to 10 years for any RIC identified resets.
So when I value the instrument in the middle of its lifecycle, I see that I need a RIC reset for the index, the RIC rule then gets applied, sees this as the only quote in the last 10 years, then uses it.

Reply:
ok, we can leave for now, this will relate to the issue of how a user sets up recipes, the current implication from all the current examples is that they need _everything_ for a call to work